music_assistant/controllers/music.py

```diff
@@ -842,10 +842,9 @@ async def get_loudness(
             },
         )
         if db_row and db_row["loudness"] != inf and db_row["loudness"] != -inf:
-            loudness = db_row["loudness"]
+            loudness = round(db_row["loudness"], 2)
             loudness_album = db_row["loudness_album"]
-            if loudness_album in (inf, -inf):
-                loudness_album = None
+            loudness_album = None if loudness_album in (None, inf, -inf) else round(loudness_album, 2)
```

Comment:
this doesn't fit a one-liner anymore. See the linter warnings from CI.

TIP: setup the dev environment with the setup script so pre-commit gets installed

Reply:
Sorry, I just accepted your suggestion in Github. I've now re-done this locally. Hope it works now.